bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt

```diff
@@ -47,6 +47,17 @@ class ModelImporter(private val root: INode) {
     private var currentNodeProgress = 0
     private val logger = KotlinLogging.logger {}
 
+    private fun doAndPotentiallyContinueErrors(block: () -> Unit) {
```

Comment:
```suggestion
    private fun doAndPotentiallyContinueOnErrors(block: () -> Unit) {
```
This should make it clearer. Still the method name is a bit long but I have a hard time coming up with a more concise name.

Reply:
The missing "on" was an oversight.